lib/lure/git.go

```diff
@@ -84,3 +85,24 @@ func (gitRepo GitRepo) LogCommitsBetween(baseRev string, secondRev string) ([]st
 	lines := strings.Split(out, "\n")
 	return append(lines[:0], lines[:len(lines)-1]...), nil
 }
+
+func (gitRepo GitRepo) GetActiveBranches() ([]string, error) {
+	out, err := gitRepo.Cmd("branch", "-r")
+	if err != nil {
+		return nil, err
+	}
+	// remove first line which is always origin/HEAD -> origin/master and last empty line (trim space)
+	branches := strings.Split(strings.TrimSpace(out), "\n")[1:]
+
+	// removing the remote prefix (origin/ most of the time)
+	for i := range branches {
+		branches[i] = strings.SplitN(branches[i], "/", 2)[1]
```

Comment:
doesn't come up so often internally since we don't fork a lot but when I do I often add a remote called upstream pointing to the original repo, in case I want to fetch from there.

doing something like a regex replace would have less assumptions: replace('origin/(stuff)', '$1')

Reply:
my concern was that it might not be named origin, but I'm too familiar with git. If you clone and don't specify the name of the remote repo I guess it will always be origin? So your regex would always work?

Reply:
You're right that in the context of lure cloning it should always be called origin. So probably your assumptions are fairly safe and what you have is okay.